com.github.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-icons: appimagepool
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - cp -R appimagepool/data /app/appimagepool/data
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - chmod +x /app/appimagepool/appimagepool
+      - chmod +x /app/appimagepool
+      - install -Dm644 appimagepool/appimagepool.svg /app/share/icons/hicolor/scalable/apps/appimagepool.svg
+      - mkdir /app/bin
+      - ln -s /app/appimagepool/appimagepool /app/bin/appimagepool
+      - install -Dm644 appimagepool/appimagepool.desktop /app/share/applications/appimagepool.desktop
+      - install -Dm644 com.github.prateekmedia.appimagepool.appdata.xml /app/share/metainfo/com.github.prateekmedia.appimagepool.appdata.xml
+    sources:
+      - type: archive
+        skip-arches: [aarch64]
```

Comment:
this is not what I meant. I need a flathub.json for that, as explained on the page I linked.

Reply:
I didn't get that, what flathub.json?

Reply:
Ok got it, fixed.